pkg/apis/planetscale/v2/vitesscluster_types.go

```diff
@@ -148,6 +152,17 @@ const (
 	ImmediateVitessClusterUpdateStrategyType VitessClusterUpdateStrategyType = "Immediate"
 )
 
+type ExternalVitessClusterUpdateStrategyOptions struct {
```

Comment:
We should add defaulting for this new sub-struct in `DefaultUpdateStrategy()`, whenever the `type` is `external`, so we can safely assume this is non-nil after checking `type`.

Reply:
Yep, done.